stl/inc/ranges

```diff
@@ -9478,6 +9755,16 @@ namespace ranges {
         }
     };
 
+#if _HAS_CXX23
+    template <class _Rng, class _Fn, size_t _Nx>
+    inline constexpr auto _Compile_time_max_size<adjacent_transform_view<_Rng, _Fn, _Nx>> =
+        _Compile_time_max_size<adjacent_view<_Rng, _Nx>>;

```

Comment:
no change requested - this could be `_Adjacent_view_compile_time_max_size<_Rng, _Nx>()` to avoid the indirection.